temporalcli/commandsmd/commands.md

```diff
@@ -377,6 +377,163 @@ Search Attribute commands enable operations for the creation, listing, and remov
 * `--name` (string[]) - Search Attribute name. Required.
 * `--yes`, `-y` (bool) - Confirm prompt to perform deletion.
 
+### temporal schedule: Perform operations on Schedules.
+
+Schedule commands allow the user to create, use, and update Schedules.
+Schedules allow starting Workflow Execution at regular times.
+
+#### Options
+
+Includes options set for [client](#options-set-for-client).
+
+### temporal schedule backfill: Backfills a past time range of actions.
+
+ The `temporal schedule backfill` command runs the Actions that would have been run in a given time
+interval, all at once.
+
+ You can use backfill to fill in Workflow Runs from a time period when the Schedule was paused, from
+before the Schedule was created, from the future, or to re-process an interval that was processed.
+
+Schedule backfills require a Schedule ID, along with the time in which to run the Schedule. You can
+optionally override the overlap policy. It usually only makes sense to run backfills with either
+`BufferAll` or `AllowAll` (other policies will only let one or two runs actually happen).
+
+Example:
+
+```
+  temporal schedule backfill           \
+    --schedule-id 'your-schedule-id'   \
+    --overlap-policy BufferAll         \
+    --start-time 2022-05-01T00:00:00Z  \
+    --end-time   2022-05-31T23:59:59Z
+```
+
+#### Options set for overlap policy:
+
+* `--overlap-policy` (string-enum) - Overlap policy. Options: Skip, BufferOne, BufferAll, CancelOther, TerminateOther, AllowAll. Default: Skip.
+
+#### Options set for schedule id:
+
+* `--schedule-id`, `-s` (string) - Schedule id. Required.
+
+#### Options
+
+* `--end-time` (timestamp) - Backfill end time. Required.
+* `--start-time` (timestamp) - Backfill start time. Required.
+
+### temporal schedule create: Create a new Schedule.
+
+The `temporal schedule create` command creates a new Schedule.
+
+Example:
+
+```
+  temporal schedule create                                    \
+    --schedule-id 'your-schedule-id'                          \
+    --calendar '{"dayOfWeek":"Fri","hour":"3","minute":"11"}' \
+    --workflow-id 'your-base-workflow-id'                     \
+    --task-queue 'your-task-queue'                            \
+    --workflow-type 'YourWorkflowType'
+```
+
+Any combination of `--calendar`, `--interval`, and `--cron` is supported.
+Actions will be executed at any time specified in the Schedule.
+
+#### Options set for schedule configuration:
+
+* `--calendar` (string[]) - Calendar specification in JSON, e.g. `{"dayOfWeek":"Fri","hour":"17","minute":"5"}`.
+* `--catchup-window` (duration) - Maximum allowed catch-up time if server is down.
+* `--cron` (string[]) - Calendar spec in cron string format, e.g. `3 11 * * Fri`.
+* `--end-time` (timestamp) - Overall schedule end time.
```

Comment:
Nice, can you document this new field type in the big xml/markdown comment near the top of this file?